Frontend/ui-driver/src/Screens/NotificationsScreen/Controller.purs

```diff
@@ -111,10 +111,30 @@ eval (NotificationDetailModelAC NotificationDetailModel.BackArrow) state =
     [ pure BackPressed
     ]
 
+eval (NotificationDetailModelAC (NotificationDetailModel.LikeMessage)) state = do
+  let likes = if state.notificationDetailModelState.likeStatus then state.notificationDetailModelState.likeCount - 1 else state.notificationDetailModelState.likeCount + 1
+  let likeStatus = not state.notificationDetailModelState.likeStatus
```

Comment:
instead using 4 let this can be done using just a single let.

Reply:
done